pages/services/edge-lb/1.1/installing/index.md

```diff
@@ -88,11 +75,7 @@ If you need to deploy a local Universe containing your own set of packages, you
     sudo make base
     ```
 
-3. Obtain the Edge-LB stub universe JSON files from the support downloads site. Note that there are two files required:
-- `stub-universe-edgelb.json`
-- `stub-universe-edgelb-pool.json`
-
-4. To add the JSON definitions to the universe, use the `add-stub-universe.sh` script.  Each run of the `add-stub-universe.sh` script will process the JSON file, generate the necessary JSON and Mustache files, and add them to `stub-repo/packages/<X>/<packagename>`.  
+3. Use `add-stub-universe.sh` script to add the JSON definitions obtained in ## Obtaining package artifacts section to the universe.  Each run of the `add-stub-universe.sh` script will process the JSON file, generate the necessary JSON and Mustache files, and add them to `stub-repo/packages/<X>/<packagename>`.
```

Comment:
Should `universe` be `Universe`? BTW, didn't we rename it to `Catalog` now?

Reply:
I have changed to `Universe`. Renaming things to `Catalog` is outside of the scope of this change as we would need to update not only Edge-LB docs as all the documents to which Edge-Lb's docs link to in order to be consistent. I believe this is a task for Docs team, which can do it in a coordinated manner.

Reply:
Change this line to this:
<p class="message--note"><strong>NOTE: </strong>You will get a "page not found" message if you attempt to download the artifacts without a current customer service account.</p>

<p class="message--note"><strong>NOTE: </strong>You will get a "page not found" message if you attempt to download the artifacts without logging in using your customer service account.</p>